tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,338 @@
+
+**Met**
+https://github.com/nodejs/node/blob/main/CODE_OF_CONDUCT.md
+
+> The project MUST clearly define and publicly document the key roles in the project and their responsibilities, including any tasks those roles must perform. It MUST be clear who has which role(s), though this might not be documented in the same way. (URL required)
+
+**Met**
+https://github.com/nodejs/node/blob/main/GOVERNANCE.md
+
+> The project MUST be able to continue with minimal interruption if any one person dies, is incapacitated, or is otherwise unable or unwilling to continue support of the project. In particular, the project MUST be able to create and close issues, accept proposed changes, and release versions of software, within a week of confirmation of the loss of support from any one individual. This MAY be done by ensuring someone else has any necessary keys, passwords, and legal rights to continue the project. Individuals who run a FLOSS project MAY do this by providing keys in a lockbox and a will providing any needed legal rights (e.g., for DNS names).
+
+**Met**
+
+> The project SHOULD have a "bus factor" of 2 or more. (URL required)
+
+**Met**
+https://github.com/nodejs/node/blob/main/README.md#current-project-team-members
+
+## Documentation
+
+> The project MUST have a documented roadmap that describes what the project intends to do and not do for at least the next year. (URL required)
+
+**Met** https://github.com/nodejs/node/blob/HEAD/doc/contributing/strategic-initiatives.md
+
+The project MUST include documentation of the architecture (aka high-level design) of the software produced by the project. If the project does not produce software, select "not applicable" (N/A). (URL required)
+
+**Met** https://github.com/nodejs/node/tree/main/doc/contributing
+
+> The project MUST document what the user can and cannot expect in terms of security from the software produced by the project (its "security requirements"). (URL required)
+
+**Met**
+https://github.com/nodejs/node/blob/main/SECURITY.md and https://github.com/nodejs/node/blob/main/SECURITY.md#the-nodejs-threat-model
+
+> The project MUST provide a "quick start" guide for new users to help them quickly do something with the software. (URL required)
+
+**Met**
+https://nodejs.dev/en/learn/
+
+
+The project MUST make an effort to keep the documentation consistent with the current version of the project results (including software produced by the project). Any known documentation defects making it inconsistent MUST be fixed. If the documentation is generally current, but erroneously includes some older information that is no longer true, just treat that as a defect, then track and fix as usual.
+
+**Met**
+
+
+The project repository front page and/or website MUST identify and hyperlink to any achievements, including this best practices badge, within 48 hours of public recognition that the achievement has been attained. (URL required) 
+
+**Met**
```

Comment:
This appears to be missing the required URL.

Reply:
Yes, good catch @tniessen. We have an issue open for it: https://github.com/nodejs/nodejs.org/issues/5432#issuecomment-1687553279. I hope that we can integrate it in the website 🤞